content/en/docs/components/serving/bentoml.md

```diff
@@ -0,0 +1,258 @@
+end-to-end model serving workflow, with DevOps best practices baked in.
+
+### Prerequisites
+
+1. a Kubernetes cluster and `kubectl` installed on your local machine.
+    * `kubectl` install instruction: https://kubernetes.io/docs/tasks/tools/install-kubectl/
+2. Docker and Docker Hub installed and configured in your local machine.
+    * Docker install instruction: https://docs.docker.com/get-docker/
+3. Python 3.6 or above and required PyPi packages: `bentoml`, `scikit-learn`
+    * ```pip install bentoml scikit-learn```
+
+### Build an iris classifier model server with BentoML
+
+The following code defines a BentoML prediction service that requires a `scikit-learn` model, and
+asks BentoML to figure out the required PyPI packages automatically. It also defined an
+API, which is the entry point for accessing this prediction service. And the API is
+expecting a `pandas.DataFrame` object as its input data.
+
+```python
+# iris_classifier.py
+from bentoml import env, artifacts, api, BentoService
+from bentoml.handlers import DataframeHandler
+from bentoml.artifact import SklearnModelArtifact
+
+@env(auto_pip_dependencies=True)
+@artifacts([SklearnModelArtifact('model')])
+class IrisClassifier(BentoService):
+
+    @api(DataframeHandler)
+    def predict(self, df):
+        return self.artifacts.model.predict(df)
+```
+
+The following code trains a classifier model and serve it with the IrisClassifier
+defined above:
+
+```python
+# main.py
+from sklearn import svm
+from sklearn import datasets
+
+from iris_classifier import IrisClassifier
+
+if __name__ == "__main__":
+    # Load training data
+    iris = datasets.load_iris()
+    X, y = iris.data, iris.target
+
+    # Model Training
+    clf = svm.SVC(gamma='scale')
+    clf.fit(X, y)
+
+    # Create a iris classifier service instance
+    iris_classifier_service = IrisClassifier()
+
+    # Pack the newly trained model artifact
+    iris_classifier_service.pack('model', clf)
+
+    # Save the prediction service to disk for model serving
+    saved_path = iris_classifier_service.save()
+```
+
+The sample code above can be found in the BentoML repository, run them directly with the
+following command:
+
+```shell
+git clone git@github.com:bentoml/BentoML.git
+python ./bentoml/guides/quick-start/main.py
+```
+
+After saving the BentoService instance, you can now start a REST API server with the
+model trained and test the API server locally:
+
+```shell
+# Start BentoML API server:
+bentoml serve IrisClassifier:latest
+```
+
+```bash
+# Send test request
+curl -i \
+  --header "Content-Type: application/json" \
+  --request POST \
+  --data '[[5.1, 3.5, 1.4, 0.2]]' \
+  localhost:5000/predict
+```
+
+BentoML provides a convenient way of containerizing the model API server with Docker. To
+create a docker container image for the sample model above:
+
+Find the file directory of the SavedBundle with `bentoml get` command, which is
```

Comment:
Suggestion: Change this paragraph to a list. The introductory sentence above the paragraph implies that there'll be a list, because it ends with a colon. Also, a list is probably easier to read in this case.